js/viz.js

```diff
@@ -240,6 +311,10 @@ const viz = {
     return node;
   },
 
+  dragStart() {
+    d3.event.subject.shadow = true;
```

Comment:
> Do we need to highlight the node even after the drag end? (This is the behaviour in the old version)

For now, no because we don't have the info section. This might change though.
Maybe file a follow up so we don't lose this comment though.